src/v/iceberg/values.h

```diff
@@ -108,13 +108,15 @@ using value = std::variant<
   std::unique_ptr<map_value>>;
 
 struct struct_value {
+    // The order of these fields must align with the corresponding struct type
+    // as defined in the schema, see `iceberg::struct_type`.
     chunked_vector<std::optional<value>> fields;
 };
 bool operator==(const struct_value&, const struct_value&);
 bool operator==(
   const std::unique_ptr<struct_value>&, const std::unique_ptr<struct_value>&);
 
-struct list_value {
+struct list_value { 
```

Comment:
Trailing ws

Reply:
That's what I get for using the horrible web UI